dataframe-geo/src/main/kotlin/org/jetbrains/kotlinx/dataframe/geo/io/write.kt

```diff
@@ -0,0 +1,58 @@
+package org.jetbrains.kotlinx.dataframe.geo.io
+
+import org.geotools.api.data.FileDataStoreFinder
+import org.geotools.api.data.SimpleFeatureStore
+import org.geotools.api.data.Transaction
+import org.geotools.geojson.feature.FeatureJSON
+import org.jetbrains.kotlinx.dataframe.geo.GeoDataFrame
+import org.jetbrains.kotlinx.dataframe.geo.geotools.toSimpleFeatureCollection
+import java.io.File
+
+
+fun GeoDataFrame<*>.writeGeoJson(path: String): Unit = writeGeoJson(File(path))
+
+fun GeoDataFrame<*>.writeGeoJson(file: File) {
```

Comment:
Please add for IO part some JUnit tests for reading/writing files close to your examples https://gist.github.com/AndreiKingsley/8eada06f174c681e953f8d939248d153

Reply:
Done